examples/ssh/opkssh/policy.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/fs"
+	"os"
+	"strings"
+
+	"github.com/openpubkey/openpubkey/pktoken"
+	"golang.org/x/exp/slices"
+)
+
+type SimpleFilePolicyEnforcer struct {
```

Comment:
we missed this because you split it out, but these should also be lower-case

Reply:
I don't agree with these sorts of changes in general because it sucks to have to change code from a package to expose functionality one may find useful, but in this case where this is an example it doesn't seem to matter one way or the other.

Reply:
I have made the change you requested.